src/main/java/com/gmail/nossr50/locale/LocaleLoader.java

```diff
@@ -85,7 +91,12 @@ else if (myLocale.length >= 2) {
                 locale = new Locale(myLocale[0], myLocale[1]);
             }
 
-            bundle = ResourceBundle.getBundle(BUNDLE_ROOT, locale);
+            Path localePath = Paths.get(mcMMO.getMainDirectory(), "locale_" + (locale != null ? locale.toString() : "") + ".properties");
+            try(Reader localeReader = Files.newBufferedReader(localePath)) {
+                bundle = new PropertyResourceBundle(localeReader);
+            } catch (IOException e) {
+                bundle = ResourceBundle.getBundle(BUNDLE_ROOT, locale);
```

Comment:
Definitely should be logged. As I am not completely familiar with mcMMO codebase yet, I left it undone for now.